### ISSUE TITLE
Can't build fuzzer/fullBench on VS in Release x64: warning treated as error

### ISSUE THREAD

**Issue by KrzysFR** [Cyan4973/zstd#169]
After doing a clean checkout of the repository, I can't build the fuzzer and fullBench VS projects because there are two warnings (see below) and they are configured with `TreatWarningAsError` set to true. Projects `zstd` and `zstdlib` are fine because the setting is set to false.

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2220	warning treated as error - no 'object' file generated	fullbench	C:\Data\Git\GitHub\zstd\lib\zstd_compress.c	221	
Warning	C4334	'<<' : result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)	fullbench	C:\Data\Git\GitHub\zstd\lib\zstd_compress.c	221	
Warning	C4334	'<<' : result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)	fullbench	C:\Data\Git\GitHub\zstd\lib\zstd_compress.c	294	
```

https://github.com/Cyan4973/zstd/blob/389648cc6d65e00abff5dca851fe3e231a2e7033/lib/zstd_compress.c#L221
```C
    const size_t hSize = 1 << params.cParams.hashLog;
```

https://github.com/Cyan4973/zstd/blob/389648cc6d65e00abff5dca851fe3e231a2e7033/lib/zstd_compress.c#L294
```C
        const size_t hSize = 1 << srcCCtx->params.cParams.hashLog;
```

Not sure about the warnings themselves, but do you require TreatWarningAsError absolutely? can they be set to false for these projects? (I can submit a PR if that's ok)

**Comment by KrzysFR** [comment]
I found a few misc things in the vcxproj as well, see #170

**Comment by Cyan4973** [comment]
Likely solved by b3a9938d0c78f213136ba841aef3240f46785bb6

**Comment by Cyan4973** [comment]
Fix within v0.6.1
